x-pack/plugins/aiops/server/routes/log_rate_analysis/analysis_handlers/top_items_handler.ts

```diff
@@ -183,15 +185,17 @@ export const topItemsHandlerFactory =
       } else if (isTextFieldCandidates(payload)) {
         const { textFieldCandidates: fieldNames } = payload;
 
-        const topCategoriesForField = await await fetchTopCategories(
+        const topCategoriesForField = await await fetchTopCategories({
```

Comment:
Is this double await still needed?

Reply:
Thanks for spotting this!! Fixed in [014746c](https://github.com/elastic/kibana/pull/189863/commits/014746c54cf557ac3fd46193c5cda8fcf7d0ea2e).